Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -338,15 +338,15 @@ public void SetRunnersExecutionLoggerConfigs()
 
         public async Task<int> RunRunsetAsync(bool doContinueRun = false)
         {
-            var result = await Task.Run(() =>
+            var result = await Task.Run(async () =>
             {
-                RunRunset(doContinueRun);
+               await RunRunset(doContinueRun);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)